src/providers/WorkflowCore.Persistence.EntityFramework/Models/PersistedExecutionPointer.cs

```diff
@@ -32,10 +32,10 @@ public class PersistedExecutionPointer
 
         public DateTime? EndTime { get; set; }
 
-        [MaxLength(100)]
+        [MaxLength(200)]
         public string EventName { get; set; }
 
-        [MaxLength(100)]
+        [MaxLength(200)]
```

Comment:
We probably need to create a migration for Postgres too?

Reply:
Added migration for PostgreSql

Reply:
Not sure about MySql if it requires local MySql server connection in order to create migration.